xls/examples/cocotb/BUILD

```diff
@@ -0,0 +1,55 @@
+# Copyright 2023 The XLS Authors
```

Comment:
what about adding the running counter as a canonical example in `examples` (or adding cocotb to an existing example?)

Reply:
Actually, I think we can create an even simpler example. What do you think about using a simple "passthrough" module that will just send back the same value you sent to it? I added this module as an example to this PR.